schemas/fingerprint-server-api.yaml

```
@@ -1768,3 +1834,67 @@ components:
       required:
         - code
         - message
+    EventUpdateRequest:
+      type: object
+      properties:
+        linkedId:
+          type: string
+          description: LinkedID value to assign to the existing event
+        tags:
+          type: object
+          description: Full tags value to be set to the existing event. Replaces any existing tags payload completely.
+        suspect:
+          type: boolean
+          description: Suspect flag indicating observed suspicious or fraudulent event
+    ErrorUpdateEvent400Response:
+      type: object
+      additionalProperties: false
+      properties:
+        error:
+          type: 'object'
+          additionalProperties: false
+          title: ErrorUpdateEvent400ResponseError
+          properties:
+            code:
+              type: string
+              description: >
+                Error code:
+                * `RequestCannotBeParsed` - the JSON content of the request contains some errors that prevented us from parsing it (wrong type/surpassed limits)
+                * `Failed` - the event is more than 10 days old and cannot be updated
+              enum:
+                - RequestCannotBeParsed
+                - Failed
+              example: 'RequestCannotBeParsed'
+            message:
+              type: string
+              description: Details about the underlying issue with the input payload
+              example: 'suspect flag must be a boolean'
+          required:
+            - code
+            - message
+    ErrorUpdateEvent409Response:
+      type: object
+      additionalProperties: false
+      properties:
+        error:
+          type: 'object'
+          additionalProperties: false
+          title: ErrorUpdateEvent409ResponseError
+          properties:
+            code:
+              type: string
+              description: >
+                Error code:
+                * `` - The event specified with request id is not ready for updates yet. Try again.
```

@petrpalata aren't you missing `StateNotReady` here? 
```suggestion
                * `StateNotReady ` - The event specified with request id is not ready for updates yet. Try again.
```